tests/parser/syntax/test_bool_ops.py

```diff
@@ -0,0 +1,23 @@
+
+def test_convert_from_bool(get_contract_with_gas_estimation):
+    code = """
+@public
+def foo() -> bool:
+    val: bool = True and True and False
+    return val
+
+@public
+def bar() -> bool:
+    val: bool = True or True or False
+    return val
+
+@public
+def foobar() -> bool:
+    val: bool = False and True or False
```

Comment:
Please add a test case for like 5 ops or something both even & odd number of ops ;)

Reply:
Sounds good will do soon! 😊👌

Reply:
Leave it, might motivate us at some point? :P